Installers/Linux/Main/mgcb.1

```diff
@@ -15,6 +15,10 @@ is a command line tool for building XNB content on Windows, Mac, and Linux deskt
 Typically it is executed by the Pipeline GUI tool when editing content or indirectly from Visual Studio or MonoDevelop during the build process of a MonoGame project. Alternatively you can use it yourself from the command line for specialized build pipelines or for debugging content processing.
 
 .TP
+.BR \fIFILENAME\fR
+Automatically selects either build or response file option for the selected file, see bellow for more detail on what these options are.
```

Comment:
below*

Reply:
No, bellow is the correct spelling.

Reply:
https://writingexplained.org/below-or-bellow-difference